compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/test/from_elements.mlir

```diff
@@ -44,3 +32,22 @@ util.func public @tensor.from_elements_0D(%arg0 : f32) -> tensor<f32> {
 // CHECK-SAME:     %[[ARG0:.+]]: f32
 //      CHECK:   %[[SPLAT:.+]] = flow.tensor.splat %[[ARG0]] : tensor<f32>
 //      CHECK:   util.return %[[SPLAT]]
+
+// -----
+
+util.func @tensor.from_elements_2D(%arg0 : f32, %arg1 : f32, %arg2 : f32, %arg3 : f32, %arg4 : f32, %arg5 : f32) -> tensor<2x3xf32> {
+  %0 = tensor.from_elements %arg0, %arg1, %arg2, %arg3, %arg4, %arg5 : tensor<2x3xf32>
+  util.return %0 : tensor<2x3xf32>
+}
+
+//      CHECK: util.func public @tensor.from_elements_2D
```

Comment:
keep checks near the lines they are checking as with the other tests in this file

Reply:
Done.